src/javaStyleGuide.md

```diff
@@ -0,0 +1,1532 @@
+% Java Style Guide
+
+::: {.NavBit}
+[« Previous](mailinglists.html) • [TOC](index.html) • [Next »](changePlanning.html)
```

Comment:
I think previous should go to codeConventions.html. Same at the bottom of this file.

Reply:
Updated.